src/main.js

```diff
@@ -72,14 +64,31 @@ function displayMiniCovers() {
   savedCoversSection.innerHTML = "";
   savedCovers.forEach(function(cover) {
   savedCoversSection.insertAdjacentHTML("afterbegin", `
-    <div class="mini-cover" data-id=${cover.id}>
-      <img class="cover-image" src=${cover.cover}>
+   <div class="mini-cover" id=${cover.id}>
+     <img class="cover-image" src=${cover.cover}>
       <h2 class="cover-title">${cover.title}</h2>
       <h3 class="tagline">A tale of <span class="tagline-1">${cover.tagline1}</span> and <span class="tagline-2">${cover.tagline2}</span></h3>
       <img class="price-tag" src="./assets/price.png">
       <img class="overlay" src="./assets/overlay.png">
     </div>`)
-  })
+  });
+}
+
+
+function deleteSelectedCover(event) {
```

Comment:
That was really good collaborative effort on both our parts.  We found the problem, and worked together to figure out the solution.  GO TEAM!